src/pages/conference/info/helpers/getConferenceByIdQuery.js

```diff
@@ -0,0 +1,32 @@
+import { gql } from 'react-apollo';
+
+const GET_CONFERENCE_BY_ID_QUERY = gql`
+  query getConferenceByID($id: ID!) {
+    getConferenceByID(id: $id) {
+      id
+      title
+      description
+      start_date
+      end_date
+      organizerDetail {
+        id
+        name
+        email
+        website
+        phone
+      }
+      coOrganizerDetails {
+        id
+        name
+        email
+        website
+        phone
+        conference {
+          id
+        }
+      }
+    }
+  }
+`;
+
+export default GET_CONFERENCE_BY_ID_QUERY;
```

Comment:
We have a graphql query here, why aren't others?

Reply:
Because I think we don't have ti split all of them to separate files, this one is required my multiple components though. we should talk about this also.